apps/backend/src/database/database.module.ts

```diff
@@ -13,6 +35,41 @@ function getSynchronize(configService: ConfigService): boolean {
   }
 }
 
+function toObject(fields: string[], values?: string[]): Record<number, string> {
+  if (values) {
+    const result: Record<number, string> = {};
+    for (let i = 0; i < values.length; i++) {
+      if (!sensitiveKeys.some((regex) => regex.test(fields[i]))) {
+        result[i] = values[i];
+      } else {
+        result[i] = '[REDACTED]';
+      }
+    }
+    return result;
+  } else {
+    return {};
+  }
+}
+
+function logQuery(
+  sql: string,
+  connection: {fields: string[]; bind: string[]; type: string}
+) {
+  if (connection.type === 'INSERT' || connection.type === 'UPDATE') {
```

Comment:
Is there possibility that sensitive parameters will be found in other connection types? What is your though on filtering for each type.